braindecode/models/deep4.py

```diff
@@ -138,6 +138,7 @@ def __init__(
             n_times=n_times,
             input_window_seconds=input_window_seconds,
             sfreq=sfreq,
+            add_log_softmax=True,
```

Comment:
it should be added to params of the model in the docstring, and used  here

Reply:
Fixed, thanks!